packages/effect/src/internal/subscriptionRef.ts

```diff
@@ -37,17 +37,18 @@ class SubscriptionRefImpl<in out A> implements SubscriptionRef.SubscriptionRef<A
     readonly pubsub: PubSub.PubSub<A>,
     readonly semaphore: Effect.Semaphore
   ) {
+    super()
     this[Readable.TypeId] = Readable.TypeId
     this[Subscribable.TypeId] = Subscribable.TypeId
-    this.get = Ref.get(this.ref)
```

Comment:
I think internally we should still use the module APIs.

Reply:
Done